client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -36,7 +36,14 @@ export default function PaymentInfoBlock( {
 	}
 
 	if ( hasPaymentMethod( purchase ) && isPaidWithCredits( purchase ) ) {
-		return <PaymentInfoBlockWrapper>{ translate( 'Credits' ) }</PaymentInfoBlockWrapper>;
+		return (
+			<PaymentInfoBlockWrapper>
+				<div className={ 'manage-purchase__no-payment-method' }>
```

Comment:
Really minor, but you don't have to jump out of JSX into javascript to add a string. You can include the string directly in the JSX.

```suggestion
				<div className="manage-purchase__no-payment-method">
```

That said, I know this was just copy/pasted from further down in this file, so you don't necessarily need to change anything. I just wanted to mention it so you would know.

Reply:
Thank you for this, I have made the changes. 🙌🏻